dig.go

```diff
@@ -450,26 +321,6 @@ func newNode(k key, ctor interface{}, ctype reflect.Type) (*node, error) {
 	}, err
 }
 
-// Retrieves the types of the arguments of a constructor in-order.
-//
-// If the constructor is a variadic function, the returned list does NOT
-// include the implicit slice argument because dig does not support passing
-// those values in yet.
-func getConstructorArgTypes(ctype reflect.Type) []reflect.Type {
-	numArgs := ctype.NumIn()
-	if ctype.IsVariadic() {
-		// NOTE: If the function is variadic, we skip the last argument
-		// because we're not filling variadic arguments yet. See #120.
-		numArgs--
-	}
-
-	args := make([]reflect.Type, numArgs)
```

Comment:
This is not needed anymore because we have already gathered this information in `newParamList`.